src/script2.js

```diff
@@ -0,0 +1,56 @@
+export function addItem(taskList, display) {
+  const newItem = document.querySelector('.new-item').value;
+  const i = taskList.length;
+  const element = {
+    description: newItem,
+    completed: false,
+    index: i,
+  };
+  taskList.push(element);
+  display(element);
+}
```

Comment:
- [ ] Tasks added to the todo list should have 1-based indexes, which means that the property `index` should start from `1`.
(i.e. if a 5th task is added to the list, the index of that task should equal to 5). Currently, it starts from `0`, please go ahead and fix that.

![Screenshot from 2021-09-30 19-21-06](https://user-images.githubusercontent.com/40503704/135598904-1d6d9844-6222-4ca5-af3b-333ad09e88d7.png)

Reply:
okay, let me correct that.

Reply:
This issue is well resolved! ✔️